add-on/panels/js/conversationAppStore.js

```diff
@@ -42,8 +42,13 @@ loop.store.ConversationAppStore = (function() {
 
     // Start listening for specific events, coming from the window object.
     this._eventHandlers = {};
-    ["unload", "LoopHangupNow", "socialFrameAttached", "socialFrameDetached", "ToggleBrowserSharing"]
-      .forEach(function(eventName) {
+    [
+      "unload",
+      "LoopHangupNow",
+      "socialFrameAttached",
+      "socialFrameDetached",
+      "ToggleBrowserSharing"
+    ].forEach(function(eventName) {
```

Comment:
I realise this is tidy up, but I don't see the point of including it in this patch, as there's nothing relating to it here.